### PR TITLE
Feature/redux setup

### PR DESCRIPTION
**Pull request by paparazzie**
@naj21 I created the service folder and the store file, let me know what else to add the the redux-setup

### CODE REVIEW

**Comment by hackthievist** [comment]
Once the comments on the preceding PRs are resolved and the PRs subsequently merged, I'd approve this one, it's currently difficult to review as there are changes from those PRs here.

File: app/src/store.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { createStore } from 'redux'
+
+export  default createStore()
```
[the remark this answers is not in the document]
Comment (paparazzie) [reply app/src/store.js:3]: you mean adding the combineReducers that would take all the reducers?

[the remark this answers is not in the document]
Comment (naj21) [reply app/src/store.js:3]: No, passing the reducers to the store, you can create an empty ./rootReducers file in the app directory and pass that in createStore(). It's in the rootReducer we'd use the combineReducers.
You also need to set up the middleware (redux-thunk).

https://redux.js.org/recipes/configuring-your-store/
Screenshot 2020-06-02 at 22 54 53

**Comment by paparazzie** [comment]
> @naj21 I created the service folder and the store file, let me know what else to add the the redux-setup

@naj21 I made the necessary adjustment and created two folders called enhancers and middleware, this is used to configure the store, I also created a rootReducer file in the services folder.

File: app/src/store.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
-import { createStore } from 'redux'
+import { createStore,applyMiddleware  } from 'redux'
```
Comment (hackthievist) [inline app/src/store.js]:
```suggestion
import { createStore, applyMiddleware  } from 'redux'
```

Is eslint activated?

Comment (paparazzie) [reply app/src/store.js]: no, can I just merge the branch that Jessica created for the eslint-setup? I mean that's the point right?

**Comment by hackthievist** [comment]
I also noticed you aren't using semi-colons and I'm fine with it. The problem would be if Jessica is using them. There needs to be consistency.

File: app/src/store.js (hunk 2 above)
Comment (hackthievist) [reply app/src/store.js]: Yes exactly. You can do that

Comment (naj21) [reply app/src/store.js]: > no, can I just merge the branch that Jessica created for the eslint-setup? I mean that's the point right?

Or you can pull develop, I have already merged the ESLint branch

**Comment by naj21** [comment]
> > @naj21 I created the service folder and the store file, let me know what else to add the the redux-setup
> 
> @naj21 I made the necessary adjustment and created two folders called enhancers and middleware, this is used to configure the store, I also created a rootReducer file in the services folder.

rootReducer should be directly under the app directory

**Comment by paparazzie** [comment]
> I also noticed you aren't using semi-colons and I'm fine with it. The problem would be if Jessica is using them. There needs to be consistency.

okay would make the changes

**Comment by paparazzie** [comment]
> > > @naj21 I created the service folder and the store file, let me know what else to add the the redux-setup
> > 
> > 
> > @naj21 I made the necessary adjustment and created two folders called enhancers and middleware, this is used to configure the store, I also created a rootReducer file in the services folder.
> 
> rootReducer should be directly under the app directory
app or src directory?

**Comment by paparazzie** [comment]
@naj21 @hackthievist please review updated changes.

**Pull request merged**